src/config/config.js

```diff
@@ -42,6 +42,9 @@ const config = {
         312: process.env.HUB_REPORT_URL_312,
         644: process.env.HUB_REPORT_URL_644,
     },
+    featureFlags: {
+        transferVerification: process.env.FEATURE_FLAG_TRANSFER_VERIFICATION !== 'false',
```

Comment:
```suggestion
        transferVerification: process.env.FEATURE_FLAG_TRANSFER_VERIFICATION === 'true',
```

Reply:
I think this is the behavior we want. Basically, transferVerification should be off by default, and it should take a very specific environment flag to turn it on.